feature-libs/cart/base/components/add-to-cart/add-to-cart.component.html

```diff
@@ -18,11 +18,18 @@
       >
     </div>
   </div>
+  <ng-container *ngIf="hasStock">
+    <ng-container *cxFeatureLevel="'5.1'">
```

Comment:
remove this feature flag, since bopis will be in 6.0

Reply:
what is this outlet for? There is `CartOutlets.ADD_TO_CART_PICKUP_OPTION"` for pickup options in line 28.